nautobot/dcim/models/locations.py

```diff
@@ -214,6 +216,34 @@ class Location(TreeNode, StatusModel, PrimaryModel):
         null=True,
     )
     description = models.CharField(max_length=200, blank=True)
+    facility = models.CharField(max_length=50, blank=True, help_text="Local facility ID or description")
+    asn = ASNField(
+        blank=True,
+        null=True,
+        verbose_name="ASN",
+        help_text="32-bit autonomous system number",
+    )
+    time_zone = TimeZoneField(blank=True)
+    physical_address = models.CharField(max_length=200, blank=True)
+    shipping_address = models.CharField(max_length=200, blank=True)
+    latitude = models.DecimalField(
+        max_digits=8,
+        decimal_places=6,
+        blank=True,
+        null=True,
+        help_text="GPS coordinate (latitude)",
+    )
+    longitude = models.DecimalField(
+        max_digits=9,
+        decimal_places=6,
+        blank=True,
+        null=True,
+        help_text="GPS coordinate (longitude)",
+    )
+    contact_name = models.CharField(max_length=50, blank=True)
+    contact_phone = models.CharField(max_length=20, blank=True)
```

Comment:
Let's consider increasing this `max_length` as well so as to support numbers like `+44 7624 800555 extension 12345`; would also make it so that our Location factory could use the phone number faker.